src/Rendering/babylon.geometryBufferRenderer.ts

```diff
@@ -3,10 +3,22 @@ module BABYLON {
      * This renderer is helpfull to fill one of the render target with a geometry buffer.
      */
     export class GeometryBufferRenderer {
+        /**
+         * Represents the index of the position texture in the G-Buffer textures array
+         */
+        public positionIndex: number = 2;

```

Comment:
@deltakosh @CraigFeldspar 
Is that ok for you to expose the textures indices like that? Depth and normal textures will always be at indices 0 and 1 so I didn't expose these indices here